packages/vue-final-modal/src/components/ModalsContainer.vue

```diff
@@ -10,6 +10,10 @@ const { modalsContainers, dynamicModals } = useVfm()
 const uid = Symbol(__DEV__ ? 'ModalsContainer' : '')
 const shouldMount = computed(() => uid === modalsContainers.value?.[0])
 
+defineOptions({
+  inheritAttrs: false,
+})
+
```

Comment:
This PR should contain only this change alone. The version updates are handled upon release/publish.

An explanation of the issue would helpful. Why does this need to be false?